docs/src/design/aux_table/pow2.md

```diff
@@ -143,13 +145,15 @@ Let $t_i$ be a value in our virtual row computed from values $a_i$ and $h$ in th
 - $t_i = a_{i-1} - a_i$ for $i \in \{1, 2, ..., 7\}$
 - $t_8 = a_7 - h$
 
-We can use selector $k_0$ to turn on the validity constraint for the first row and selector $k_1$ to apply the transition constraint to all but the last row in the cycle. This gives us the following constraints.
+We can use selector $k_0$ to turn on the constraint for the first row of $zp$ and selector $k_1$ to apply the transition constraint to all but the last row in the cycle. This gives us the following constraints.
 
-$$k_0 \cdot \left(z - (\sum\limits_{i=0}^8 t_i \cdot 2^i)\right) = 0$$
+$$k_0 \cdot (zp - 0) = 0$$
```

Comment:
nit: subtracting zero is not necessary - It's fine to do $$k_0 \cdot zp = 0$$

Reply:
Done. Please see [bfa8a00](https://github.com/maticnetwork/miden/pull/232/commits/bfa8a0019bce0ea03c28f1d1f5e83672e00b4066).